flow/designs/asap7/mock-array/metadata-base-ok.json

```diff
@@ -15,59 +15,59 @@
   "cts__design__die__area": 133987,
   "cts__design__die__area__post_repair": 133987,
   "cts__design__die__area__pre_repair": 133987,
-  "cts__design__instance__area": 119951,
+  "cts__design__instance__area": 121339,
   "cts__design__instance__area__macros": 119439,
   "cts__design__instance__area__macros__post_repair": 119439,
   "cts__design__instance__area__macros__pre_repair": 119439,
   "cts__design__instance__area__post_repair": 119918,
   "cts__design__instance__area__pre_repair": 119918,
-  "cts__design__instance__area__stdcell": 511.408,
+  "cts__design__instance__area__stdcell": 1899.57,
   "cts__design__instance__area__stdcell__post_repair": 478.968,
   "cts__design__instance__area__stdcell__pre_repair": 478.647,
-  "cts__design__instance__count": 10619,
-  "cts__design__instance__count__hold_buffer": 445.0,
+  "cts__design__instance__count": 29661,
+  "cts__design__instance__count__hold_buffer": 19487.0,
```

Comment:
It's interesting that this went up so much...

Reply:
LOTS of hold cells at the edges of mock-array. I wonder if carefully articulated .sdc files can do 500ps...

I wonder if reducing https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/077cb25732f7162ac688f0944aa5020d32c5af42/flow/designs/asap7/mock-array/constraints.sdc#L8 would improve things.